src/epgsql_sock.erl

```diff
@@ -140,25 +149,37 @@ handle_call({standby_status_update, FlushedLSN, AppliedLSN}, _From,
     send(State, ?COPY_DATA, epgsql_wire:encode_standby_status_update(ReceivedLSN, FlushedLSN, AppliedLSN)),
     {reply, ok, State#state{repl_last_flushed_lsn = FlushedLSN, repl_last_applied_lsn = AppliedLSN}};
 
+handle_call({timed, Command, Timeout}, From, State) ->
+  Timer = erlang:start_timer(Timeout, self(), timed_command),
```

Comment:
I think if we decide to have a single timer, it may be reasonable to check if there is already active timer and cancel it (probably with `error_logger` warning). Or check which one will fire earlier and choose it.